chia/daemon/client.py

```
@@ -46,7 +61,9 @@ async def _get(self, request: WsRpcMessage) -> WsRpcMessage:
         request_id = request["request_id"]
         self._request_dict[request_id] = asyncio.Event()
         string = dict_to_json_str(request)
-        asyncio.create_task(self.websocket.send(string))
+        if self.websocket is None:
```

Won't the `asyncio.create_task(self.websocket.send(string))` throw anyway in this case?

---

why would it throw?

---

if `self.websocket is none` - then I would expect `asyncio.create_task(self.websocket.send(string))` to throw so one approach is to not bother with the "if" check and just let it throw. Which I assume the previous code does.

---

I'll just do that.

---

correction, one of the linters wants it.